property.go

```diff
@@ -3,15 +3,15 @@ package opslevel
 import "fmt"
 
 type PropertyDefinitionInput struct {
-	Name   string     `json:"name" defaults:"John Doe"`
-	Schema JSONString `json:"schema" defaults:"{\"$schema\":\"https://json-schema.org/draft/2020-12/schema\",\"title\":\"Packages\",\"description\":\"A list of packages.\",\"type\":\"array\",\"items\":{\"type\":\"object\",\"properties\":{\"name\":{\"type\":\"string\"},\"version\":{\"type\":\"string\"},\"lock_file\":{\"type\":\"string\"},\"manager\":{\"type\":\"string\"},},\"required\":[\"name\",\"version\"],},\"minItems\":0,\"uniqueItems\":true}"`
+	Name   string     `json:"name" default:"John Doe"`
+	Schema JSONString `json:"schema,string" default:"\"{\\\"$schema\\\":\\\"https://json-schema.org/draft/2020-12/schema\\\",\\\"title\\\":\\\"Packages\\\",\\\"description\\\":\\\"A list of packages.\\\",\\\"type\\\":\\\"array\\\",\\\"items\\\":{\\\"type\\\":\\\"object\\\",\\\"properties\\\":{\\\"name\\\":{\\\"type\\\":\\\"string\\\"},\\\"version\\\":{\\\"type\\\":\\\"string\\\"},\\\"lock_file\\\":{\\\"type\\\":\\\"string\\\"},\\\"manager\\\":{\\\"type\\\":\\\"string\\\"},},\\\"required\\\":[\\\"name\\\",\\\"version\\\"],},\\\"minItems\\\":0,\\\"uniqueItems\\\":true}\""`
 }
 
 type PropertyDefinition struct {
-	Aliases []string `graphql:"aliases" json:"aliases" defaults:"-"`
-	Id      ID       `graphql:"id" json:"id" defaults:"-"`
-	Name    string   `graphql:"name" json:"name" defaults:"-"`
-	Schema  JSON     `json:"schema" scalar:"true" defaults:"-"` // Do not add graphql struct tag here
+	Aliases []string `graphql:"aliases" json:"aliases" default:"-"`
+	Id      ID       `graphql:"id" json:"id" default:"-"`
+	Name    string   `graphql:"name" json:"name" default:"-"`
+	Schema  JSON     `json:"schema" scalar:"true" default:"-"` // Do not add graphql struct tag here
```

Comment:
Blocking

Can we remove this comment too at the end of the line too? It shows up in the godoc for the `Schema` field which is unnecessary and confusing.